ReactViewControl/ReactViewControl.csproj

```diff
@@ -34,6 +34,7 @@
   <ItemGroup>
     <Reference Include="PresentationCore" />
     <Reference Include="PresentationFramework" />
+    <Reference Include="System.Web.Extensions" />
```

Comment:
we should not add another dependency just because of a debug capability

Reply:
makes sense, if we remove the IModuleDependenciesProvider implementation from here - as suggested bellow- this won't be needed.